bigflow/commons.py

```diff
@@ -4,11 +4,11 @@
 import logging
 
 from pathlib import Path
-from datetime import datetime
-
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_JOB_EXECUTION_TIMEOUT = None
```

Comment:
Do we really need this const?
IMO it is not the best place for it anyway.

Reply:
Removed